packages/dapp/src/components/utility/FormElements.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+import styled from "styled-components";
+
+export const StyledFormContainer = styled.div`
+  margin: 1em 0;
+`;
+
+export const FormValidationMessage = styled.div`
+  color: #c00;
+  font-weight: bold;
+`;
+
+export const FormGroup = styled.div`
+  margin: 0 0 1em;
+`;
+
+export interface InputElementProps {
+  type: string;
+  name?: string;
+  value?: string;
+  readOnly?: boolean;
+  onChange?(event: any): void;
+  validate?(event: any): void;
+}
+
+export class InputElement extends React.Component<InputElementProps> {
+  constructor(props: any) {
+    super(props);
+  }
+
+  public render(): JSX.Element {
+    return (
+      <input
+        type={this.props.type}
+        name={this.props.name}
+        value={this.props.value}
+        readOnly={this.props.readOnly}
+        onChange={this.onChange.bind(this)}
```

Comment:
lets be consistent with how we bind in components.  used the `private boundFunction = () => {}` above

Reply:
Good catch